lib/vagrant-libvirt/action/create_domain.rb

```diff
@@ -198,7 +198,7 @@ def call(env)
                 path: disk[:absolute_path],
                 capacity: disk[:size],
                 owner: storage_uid(env),
-                group: storage_uid(env),
+                group: storage_gid(env),
```

Comment:
This looks to be a very straight forward fix, can you look at just sending this part, because I think changing the file modes will cause concern over security.

Reply:
Split into #1275